README.md

```diff
@@ -0,0 +1 @@
+This is a most excellent README
```

Comment:
This spelling looks incredible! I think it could benefit from some stars ✨

Reply:
noted! Thanks for the feedback. I agree that this README is not snazzy enough.